### ISSUE TITLE
How to execute the bytecode?

### ISSUE THREAD

**Issue by Chr1s-T** [ethereum/py-evm#1945]
If I deploy a contract with Solidity, and get its bytecode. Can I execute the bytecode with py-evm?

**Comment by cburgdorf** [comment]
Yes, that should be possible. I'm afraid we are lacking a good example in our docs. I will admit that we should add one to our cookbook.

Maybe this test can serve as a good starting point for your further investigations?

https://github.com/ethereum/py-evm/blob/8b26d16270acbf06d988de39cc6671cb527726be/tests/core/opcodes/test_opcodes.py#L1041-L1127
